ui/ducks/confirm-alerts/confirm-alerts.ts

```diff
@@ -24,6 +24,12 @@ export type Alert = {
    */
   key: string;
 
+  /**
+   * Whether the alert is a blocker and un-acknowledgeable, preventing the user
+   * from proceeding and relying on actions to proceed. The default is `false`.
+   */
+  isBlocking?: boolean;
```

Comment:
food for thought: an alternative name would be to use isConfirmRequired or isAcknowledgementRequired. isBlocking seems good to me.